packages/node_modules/pouchdb-adapter-http/src/index.js

```diff
@@ -173,6 +173,8 @@ function HttpPouch(opts, callback) {
       var str = nAuth.username + ':' + nAuth.password;
       var token = btoa(unescape(encodeURIComponent(str)));
       options.headers.set('Authorization', 'Basic ' + token);
+    } else if (opts.jwt_token) {
```

Comment:
This could easily be set by using `fetch` property, and I'd advice to use such property since allows you more freedom to do  operations (i.e. the token expired, and you have a refresh token to refresh credentials before proceeding with the fetch)